### DISCUSSION TITLE
PAPI Placeholder - Island Total Worth Format Rank

### DISCUSSION THREAD

**Discussion by Ramzie55** [BG-Software-LLC/SuperiorSkyblock2#415] [Implemented Feature Requests] [answered]
### PAPI Placeholder - Island Total Worth Format Rank

I was looking at the placeholders and noticed there's **%superior_island_top_worth_value_<#>%**, but there isn't a **%superior_island_total_worth_format_value_<#>%**, which is what I'm suggesting because the formatted placeholder looks a lot better.

**Comment by Pikaronga** [comment]
This would be very easy to add & make the plugin a lot better. Vouch!

**Comment by OmerBenGera** [comment, marked as answer]
For anyone looking for a solution, use `%superior_island_total_worth_format%`
